_data/retail.yml

```
@@ -199,16 +232,22 @@ websites:
     img: homedepot.png
     twitter: HomeDepot
     facebook: homedepot
+    regions:
+      - us
 
   - name: Hudson's Bay
     url: https://www.thebay.com
     img: bay.png
     twitter: hudsonsbay
+    regions:
+      - us
```

```suggestion
      - ca
```
Hudson’s Bay is a Canadian retailer that only ships domestically.